_layouts/single.html

```diff
@@ -46,7 +46,9 @@
 
       <section class="page__content" itemprop="text">
         {{ content }}
-
+        {% if page.slidesurl %}
+          <p style="font-size: smaller"><a href="{{ page.slidesurl }}">Slides</a></p>
+        {% endif %}
```

Comment:
This should be after or adjacent to the link to download the paper, not before the citation.